components/admin_console/admin_definition.jsx

```diff
@@ -5338,6 +5338,17 @@ const AdminDefinition = {
                     //     placeholder: t('admin.experimental.replyToAddress.example'),
                     //     placeholder_default: 'E.g.: "reply-to@example.com"',
                     // },
+                    {
+                        type: Constants.SettingsTypes.TYPE_BOOL,
+                        key: 'ExperimentalSettings.EnableSharedChannels',
+                        label: t('admin.experimental.shared_channels.title'),
+                        label_default: 'Shared Channels:',
+                        help_text: t('admin.experimental.shared_channels.desc'),
+                        help_text_default: 'Toggles Shared Channels',
+                        help_text_markdown: false,
+                        isHidden: it.not(it.licensed),
```

Comment:
Are we OK here to just be licensed for E10 or should it check specifically for shared channels? I'm not familiar with the licensing details of this new feature. 

```suggestion
                        isHidden: it.not(it.licensedForFeature("SharedChannels")),
```

Reply:
You are 💯 % right, it needs to be only enabled under E20